### ISSUE TITLE
Expose guess in the public API

### ISSUE THREAD

**Issue by fasiha** [RayDeCampo/nodejs-xirr#5]
Superb library!

I ran into some convergence problems, and other than increasing the maximum number of iterations for the underlying Newton-Raphson method, it was helpful for me to provide a `guess` instead of letting the library pick it. It'd be nice if users could specify a guess, like spreadsheet applications allow.

I'm happy to open a pull request.

**Comment by RayDeCampo** [comment]
A pull request would be welcome, thanks @fasiha

**Comment by jarkko** [comment]
Just to add my two cents: the function never converges with the following values, which, uh, are real numbers for a real fund for the given time period (thanks, Corona!):

```
[
  [
    "2020-03-04",
    -713.07
  ],
  [
    "2020-03-17",
    555.33
  ]
]
```

**Comment by RayDeCampo** [comment]
@jjarko, thanks for the feedback.  However, it would be better if you submitted a separate issue instead of adding on to an existing issue.

**Comment by fasiha** [comment]
@RayDeCampo thanks for the fix! Quick question: can you publish a new version 1.0.4 with the fix to npm?

**Comment by RayDeCampo** [comment]
@fasiha, just published v1.1.0 (see https://www.npmjs.com/package/xirr)
